templates/taxi/car_detail.html

```diff
@@ -19,4 +19,6 @@ <h1>Drivers</h1>
       <li>{{ driver.username }} ({{ driver.first_name }} {{ driver.last_name }})</li>
     {% endfor %}
   </ul>
+
+  <a href="{% url 'taxi:car-assign' pk=car.id %}" class="btn btn-primary">Assign me to this Car</a>
```

Comment:
Consider adding a 'Cancel' button that leads to the previous page the user was on, as per the checklist requirement: 'Add `Cancel` button apart from `Delete` one. The `Cancel` button will lead to the previous page the user was on.'